src/main/java/uk/ac/ebi/embl/api/service/MasterEntryService.java

```diff
@@ -67,8 +68,11 @@ private Entry getMasterEntryFromSubmittedXml(SubmissionOptions options) throws V
         if (masterEntry != null && StringUtils.isNotBlank(masterEntry.getComment().getText())) {
             StringWriter strWriter = new StringWriter();
             try {
+                // Wrap the comment line using the default optimal line length.
+                // Force wrapping is done in the CCWriter.
+                // TODO: we should have a unit test for the comment contructed from the XML
                 FlatFileWriter.writeBlock(strWriter, "", "", masterEntry.getComment().getText(),
-                        WrapType.EMBL_WRAP, WrapChar.WRAP_CHAR_BREAK, EmblPadding.CC_PADDING.length());
+                       WrapChar.WRAP_CHAR_SPACE, WrapType.EMBL_WRAP, EmblPadding.CC_PADDING.length(), true, null);
```

Comment:
- Do we have a unit test for XML -> comment creation (we need one to make sure we don't change existing behavior)?
- The code does not appear to do what the comment is saying because forceLineBreak is 'true'. Based on our previous discussion the forceLineBreak should be 'false'.